filters/colorization/ColorizationFilter.hpp

```diff
@@ -39,12 +39,23 @@
 
 #include <boost/array.hpp>
 
```

Comment:
Clang handles GCC diagnostics just fine.  I would think they'd be silently ignored by MSVC.  But like Howard said, I thought we were pitching this.